src/tms_service.ts

```diff
@@ -374,33 +374,45 @@ export class TMSService extends AbstractEmsService {
 
   async getMinZoom(format = 'vector'): Promise<number | undefined> {
     switch (format) {
-      case 'vector':
+      case 'vector': {
         const { sources } = (await this._getVectorStyleJsonInlined()) || { sources: {} };
         return Math.min(
           ...Object.values(sources)
-            .map(({ minzoom }) => minzoom)
```

Comment:
code change from typescript upgrade, `minzoom` was not inferred from the parameter and needed to be explicitly checked.